library/kotlin/src/io/envoyproxy/envoymobile/Element.kt

```diff
@@ -0,0 +1,16 @@
+package io.envoyproxy.envoymobile
+
+import java.util.regex.Pattern
+
+/**
+ * Element for stats.
+ */
+class Element(val element: String) {
+  init {
+    if (!Pattern.compile("^[A-Za-z_]+\$").matcher(element).matches()) {
+      throw IllegalArgumentException(
+        "Element values must conform to the regex /^[A-Za-z_]+$/"
```

Comment:
I'd use a constant here so you don't have to replicate the regex twice (and so they don't deviate over time)

Reply:
clever